client/layout/logged-out.jsx

```diff
@@ -22,7 +22,8 @@ const LayoutLoggedOut = ( {
 		[ 'is-group-' + section.group ]: !! section,
 		[ 'is-section-' + section.name ]: !! section,
 		'focus-content': true,
-		'has-no-sidebar': true // Logged-out never has a sidebar
+		'has-no-sidebar': true, // Logged-out never has a sidebar
+		'wp-singletree-layout': primary,
```

Comment:
Hm. To me it seems a tad more legible, as I don't expect the actual object contents to be of any relevance when seeing the `!!`. More consistent with https://github.com/Automattic/wp-calypso/pull/4992/files#diff-27ba10a603b45adc7e189a25058de88bL22 and L23 anyway...